src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -217,4 +196,39 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
             this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(methodId)),
         ]);
     }
+
+    private _buttonClickEvent(methodId: string): (event: Event) => Promise<InternalCheckoutSelectors> {
+
+        return (event: Event) => {
+            if (event) {
+                event.preventDefault();
+            }
+
+            if (!methodId || !this._googlePayOptions) {
+                throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+            }
+
+            const {
+                onError = () => {},
+                onPaymentSelect = () => {},
+            } = this._googlePayOptions;
+
+            return this._store.dispatch(this._paymentStrategyActionCreator.widgetInteraction(async () => {
+                return await this._displayWallet(methodId, onPaymentSelect, onError);
```

Comment:
this function is a one liner, so maybe it reads better without the curly braces?
```js
return this._store.dispatch(
  this._paymentStrategyActionCreator.widgetInteraction(
    async () => await this._displayWallet(methodId, onPaymentSelect, onError),
   { methodId }
  ),
  { queueId: 'widgetInteraction' }
);